overlord/configstate/configcore/corecfg.go

```diff
@@ -64,6 +71,13 @@ func validateExperimentalSettings(tr Conf) error {
 }
 
 func Run(tr Conf) error {
+	// check if the changes
+	for _, k := range tr.Changes() {
+		if !supportedConfigurations[k] {
+			return fmt.Errorf("cannot set %q: unsupported core config option", k)
```

Comment:
"cannot set %q: unsupported system option"

As a side note, this message isn't showing anywhere else. Is this tested?